core/node/eigenda_proxy/Cargo.toml

```diff
@@ -17,11 +17,14 @@ tracing.workspace = true
 rlp.workspace = true
 rand.workspace = true
 sha3.workspace = true
+zksync_config.workspace = true
 # we can't use the workspace version of prost because
 # the tonic dependency requires a hugher version.
 prost = "0.13.1"
-tonic = { version = "0.12.1", features = ["tls", "channel", "tls-roots"]}
+tonic = { version = "0.12.1", features = ["tls", "channel", "tls-native-roots"]}
```

Comment:
What's the reason behind this change? 🤔

Reply:
It is deprecated
```
tls-roots: Deprecated. An alias to tls-native-roots feature.
tls-native-roots: Adds system trust roots to [rustls](https://docs.rs/rustls)-based gRPC clients using the [rustls-native-certs](https://docs.rs/rustls-native-certs) crate. Not enabled by default.
```
From https://docs.rs/tonic/latest/tonic/index.html